modules/fedimint-wallet-tests/tests/tests.rs

```diff
@@ -58,14 +58,15 @@ async fn peg_in<'a>(
     assert_eq!(balance_sub.ok().await?, sats(0));
 
     let (op, address) = client.get_deposit_address(valid_until).await?;
+    info!(?address, "Peg-in address generated");
     let (_proof, tx) = bitcoin
         .send_and_mine_block(&address, bsats(PEG_IN_AMOUNT_SATS))
         .await;
     let height = dyn_bitcoin_rpc
         .get_tx_block_height(&tx.txid())
         .await?
         .context("expected tx to be mined")?;
-    info!(?height, "Peg-in transaction mined");
+    info!(?height, ?tx, "Peg-in transaction mined");
```

Comment:
Is this going to be ... longish? Should we use one of the abbreviation wrappers?

Reply:
This isn't as easy as it appears, maybe the `Debug` impl of `Transaction` should do that automatically? Or at least impl `AbbreviatedDebug`?